frontend/src/components/common/form/Form.tsx

```diff
@@ -0,0 +1,33 @@
+import './Form.scss';
+
+import React from 'react';
+import { Form as FormBase } from 'react-bootstrap';
+import { useFormikContext } from 'formik';
+
+type FormProps = {
+  /** Adds a custom class to the <Form> component */
+  className?: string;
+  children?: React.ReactNode;
+};
+
+/**
+ * Formik-connected form control that provides thin wrappers for React-Bootstrap form controls;
+ * e.g. <Form.Row>, <Form.Group>, <Form.Control>, etc.
+ */
+const Form = ({ className, children, ...rest }: FormProps) => {
+  const { handleSubmit } = useFormikContext();
+  return (
+    <FormBase noValidate className={className} onSubmit={handleSubmit} {...(rest as any)}>
```

Comment:
Can you just go `{...any}`?

Reply:
Nope. The `as any` is to "turn off" the types inferred by typescript. This is a common pattern in React where creating a custom typing when you are "extending" a base class (like here we are wrapping the react-bootstrap <Form>). It's fine as in JavaScript the types are removed and it ends up being `{...rest}` which is what we want